### PR TITLE
[ADD] website_unsaleable_options

### PR DESCRIPTION
**Pull request by eLBati**
This module prevents optional products to be sold singularly.
That is, they can only be sold if associated to their main product

### CODE REVIEW

File: website_unsaleable_options/controllers/main.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        cr.execute(
+            "select src_id from product_optional_rel where dest_id"
+            " = %s"
+            % product_id)
+        res = cr.fetchall()
+        if res:
+            products = template.browse(
+                cr, uid, [item[0] for item in res], context)
+            prod_list = [p.name for p in products]
+            message = ("You can't direcly add to cart an optional "
```
Comment (pedrobaeza) [inline website_unsaleable_options/controllers/main.py]: Translate the message.

File: website_unsaleable_options/controllers/main.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+class website_unsaleable_options(website_sale_options):
+
+    @http.route()
+    def modal(self, product_id, **kw):
+        cr, uid, context, pool = (
+            request.cr, request.uid, request.context, request.registry)
+        template = pool['product.template']
+        cr.execute(
+            "select src_id from product_optional_rel where dest_id"
```
Comment (pedrobaeza) [inline website_unsaleable_options/controllers/main.py]: Can you avoid SQL here?

**Comment by eLBati** [comment]
@pedrobaeza done, thanks

**Comment by pedrobaeza** [comment]
Maybe _website_sale_isolated_accesory_unsaleable_ can be more descriptive, but a longer. What you prefer.

**Comment by eLBati** [comment]
@pedrobaeza `website_unsaleable_isolated_options`?

**Comment by pedrobaeza** [comment]
I would keep _website_sale_ prefix to mean that it's one complement for that part. _website_sale_unsaleable_options_ seems good enough for me.

**Comment by eLBati** [comment]
@pedrobaeza done

**Comment by pedrobaeza** [comment]
:+1: 

Thanks

**Comment by hbrunn** [comment]
:+1:

**Pull request merged**